conda_smithy/templates/azure-pipelines-win.yml.tmpl

```diff
@@ -0,0 +1,94 @@
+jobs:
+- job: win_64
+  pool:
+    vmImage: vs2017-win2016
+  strategy:
+    maxParallel: 4
+    matrix:
+    {%- for config, platform in configs | sort %}
+    {%- if platform.startswith('win') %}
+      {{ config }}:
+        CONFIG: {{ config}}
+    {%- endif %}
+    {%- endfor %}
+  steps:
+    # TODO: Fast finish on azure pipelines?
+    - script: |
+        {{ fast_finish }}
+
+    - script: |
+        choco install vcpython27 -fdv -y --debug
+      condition: contains(variables['CONFIG'], 'vs2008')
+      displayName: Install vcpython27.msi (if needed)
+
+    # Cywing's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
```

Comment:
```suggestion
    # Cygwin's git breaks conda-build. (See https://github.com/conda-forge/conda-smithy-feedstock/pull/2.)
```

Reply:
That's sadly a typo that has been copied around for a while. Fixing with PR ( https://github.com/conda-forge/conda-smithy/pull/921 ).